isis/tests/data/isisimport/1M189529263EFF64KCP2977M2F1_cropped.IMG

```diff
@@ -0,0 +1,384 @@
+PDS_VERSION_ID                 = PDS3
+RECORD_TYPE                    = FIXED_LENGTH
+RECORD_BYTES                   = 2048
+FILE_RECORDS                   = 1043
+LABEL_RECORDS                  = 12
+^IMAGE_HEADER                  = 13
+^IMAGE                         = 10
+DATA_SET_ID                    = MER1-M-MI-2-EDR-SCI-V1.0
+DATA_SET_NAME                  = "MER 1 MARS MICROSCOPIC IMAGER SCIENCE EDR VERSION 1.0"
+COMMAND_SEQUENCE_NUMBER        = 1
+FRAME_ID                       = MONO
+FRAME_TYPE                     = MONO
+GEOMETRY_PROJECTION_TYPE       = RAW
+IMAGE_ID                       = 129770201
+IMAGE_TYPE                     = REGULAR
+INSTRUMENT_HOST_ID             = MER1
+INSTRUMENT_HOST_NAME           = "MARS EXPLORATION ROVER 1"
+INSTRUMENT_ID                  = MI
+INSTRUMENT_NAME                = "MICROSCOPIC IMAGER"
+INSTRUMENT_SERIAL_NUMBER       = 110
+INSTRUMENT_TYPE                = "IMAGING CAMERA"
+INSTRUMENT_VERSION_ID          = FM
+LOCAL_TRUE_SOLAR_TIME          = 12:07:38
+MAGNET_ID                      = NULL
+MISSION_NAME                   = "MARS EXPLORATION ROVER"
+MISSION_PHASE_NAME             = "PRIMARY MISSION"
+OBSERVATION_ID                 = 3204473856
+PLANET_DAY_NUMBER              = 691
+PRODUCER_INSTITUTION_NAME      = "U.S. GEOLOGICAL SURVEY"
+PRODUCT_CREATION_TIME          = 2006-01-06T22:16:21
+PRODUCT_ID                     = 1M189529263EFF64KCP2977M2F1
+PRODUCT_VERSION_ID             = "V3.0 D-22846"
+RELEASE_ID                     = 1
+ROVER_MOTION_COUNTER           = (64, 472, 321, 1798, 1331)
+ROVER_MOTION_COUNTER_NAME      = (SITE, DRIVE, IDD, PMA, HGA)
+SEQUENCE_ID                    = p2977
+SEQUENCE_VERSION_ID            = 2
+SOLAR_LONGITUDE                = 350.592
+SPACECRAFT_CLOCK_CNT_PARTITION = 1
+SPACECRAFT_CLOCK_START_COUNT   = 189529263.621
+SPACECRAFT_CLOCK_STOP_COUNT    = 189529263.774
+START_TIME                     = "2006-01-03T02:59:18.555Z"
+STOP_TIME                      = "2006-01-03T02:59:18.707Z"
+TARGET_NAME                    = MARS
+TARGET_TYPE                    = PLANET
+APPLICATION_PROCESS_ID         = 29
+APPLICATION_PROCESS_NAME       = MI
+APPLICATION_PROCESS_SUBTYPE_ID = 1
+EARTH_RECEIVED_START_TIME      = "2006-01-06T22:13:45.946Z"
+EARTH_RECEIVED_STOP_TIME       = "2006-01-06T22:14:04.177Z"
+EXPECTED_PACKETS               = N/A
+PACKET_MAP_MASK                = N/A
+RECEIVED_PACKETS               = N/A
+SPICE_FILE_NAME = /oss/merb/ops/ops/surface/strategic/ref/naif/misc/chronos/chronos.mer
```

Comment:
We can't have full paths in files anymore. If the beginning slash is changed to a space, not deleted, then all should be good and we meet our requirements